lib/distribute.c

```diff
@@ -494,20 +547,33 @@ int config_show_distribute(struct vty *vty)
 }
 
 /* Configuration write function. */
-int config_write_distribute(struct vty *vty)
+int config_write_distribute(struct vty *vty, const char *vrfname)
```

Comment:
please add a `distribute_ctx` parameter in the beginning and remove the `vrfname` parameter